doc: exceptions (#11)
ems-project/maintainers: doc/coding_standards.md

composer phpstan
`````
In order to run at level 8 for every subproject, we generated a baseline where needed.
However, it is forbidden to add new exceptions in the baseline with new Pull Requests. It is encouraged to resolve small issues on every PR so that we can disable the baseline everywhere!

## Exceptions

> I there is an unexpected exception, an exception that should never append, like an exception throw in order to solve a phpstan issue, throw a ```\RuntimeException```. We should avoid using ```\Exception``` in order to simplify logs analysis.
> 

Reviewer: ```suggestion
When there is an unexpected exception, an exception that should never happened, like an exception throw in order to solve a phpstan issue, throw a ```\RuntimeException```. We should avoid using ```\Exception``` in order to simplify logs analysis.

```
